src/errorMessages.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable max-len */
+const errorMessages = {
```

Comment:
Since there is no logic here, it would be possible to use JSON instead of JS

Reply:
I moved caseNames to a separate file, so I changed logic in errorMessages a little.